### ISSUE TITLE
html-include javascript broken

### ISSUE THREAD

**Issue by ewdurbin** [pypa/warehouse#7648]
**Describe the bug**

The javascript that powers our client side includes seems to be broken

**Expected behavior**

Current user indicator, public email, and other similar features powered by html-include.js should render.

**To Reproduce**

Login to PyPI, note that the banner in upper right is not rendered in "logged in" state.

I don't see any requests being made for the /_includes at all.

**Comment by jezdez** [comment]
I'm wondering if the CSP error is the cause for this?
![](https://dsc.cloud/a46d84/Lppy9J1UEHW9FUgoEdDJ/Screen-Shot-2020-03-26-14-02-46.56.png)

**Comment by ewdurbin** [comment]
I'm not seeing a similar CSP error locally or on PyPI at the moment.

**Comment by ewdurbin** [comment]
Full contents of the resulting `warehouse.js` bundle currently doesn't look promising and indicates that it is likely a full failure to bundle JS ala #7649 

```
!function(t) {
    var e = {};
    function u(n) {
        if (e[n])
            return e[n].exports;
        var r = e[n] = {
            i: n,
            l: !1,
            exports: {}
        };
        return t[n].call(r.exports, r, r.exports, u), r.l = !0, r.exports
    }
    u.m = t, u.c = e, u.d = function(n, r, t) {
        u.o(n, r) || Object.defineProperty(n, r, {
            configurable: !1,
            enumerable: !0,
            get: t
        })
    }, u.n = function(n) {
        var r = n && n.__esModule ? function() {
            return n.default
        } : function() {
            return n
        };
        return u.d(r, "a", r), r
    }, u.o = function(n, r) {
        return Object.prototype.hasOwnProperty.call(n, r)
    }, u.p = "/static/js/", u(u.s = 0)
}({
    0: function(n, r, t) {
        n.exports = t(null)
    },
    null: function(n, r, t) {
        var c = t(null),
            f = t(null),
            i = t(null);
        n.exports = function(n) {
            var r = c(n),
                t = f.f;
            if (t)
                for (var e, u = t(n), o = i.f, l = 0; u.length > l;)
                    o.call(n, e = u[l++]) && r.push(e);
            return r
        }
    }
});
//# sourceMappingURL=warehouse.js.d7d35cf7.map

```

**Comment by di** [comment]
Resolved by #7650.
